test/e2e/matching.cy.js

```diff
@@ -0,0 +1,30 @@
+describe('Matching', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/');
+  });
+
+  it('should display the matching component', function () {
+    const matchingComponents = this.data.components.filter((component) => component._component === 'matching')
+    matchingComponents.forEach((matchingComponent) => {
```

Comment:
```suggestion
    matchingComponents.forEach(matchingComponent => {
```

These brackets aren't needed, please remove them from fat-arrow functions.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax